model/src/main/java/com/epam/esm/model/parameters/TableName.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.esm.model.parameters;
+
+public class TableName {
```

Comment:
From constants naming point of view, usually better to name from top to bottom, from the most common to specific. In this case: `TABLE_TAG`, `TABLE_CERTIFICATE`. In a code editor easier to find the appropriate one using only `LEVEL` prefix with an autocomplete option.

Reply:
done